src/OpenLoco/Graphics/Gfx.cpp

```diff
@@ -1580,12 +1580,12 @@ namespace OpenLoco::Gfx
 
     void drawImageSolid(Gfx::RenderTarget& rt, const Ui::Point& pos, const ImageId& image, PaletteIndex_t paletteIndex)
     {
-        uint8_t palette[256];
-        memset(palette, paletteIndex, 256);
+        PaletteMapBuffer<256> palette;
+        std::fill(palette.begin(), palette.end(), 0xFF);
```

Comment:
This is incorrect should be setting it as paletteIndex

Reply:
Damn good catch, the argument order of messet irritated me here.